django/core/files/uploadhandler.py

```diff
@@ -127,6 +127,14 @@ def upload_complete(self):
         """
         pass
 
+    def upload_interrupted(self):
```

Comment:
`upload_stop` is likely a better name for these reasons:

1. It's more coherent with `file_complete` and `upload_complete` (present tense)
2. It's more coherent with the `StopUpload` exception name.

Reply:
Cool, happy to change the name.

Reply:
Bikeshedding, but I think `upload_stopped` is more similar: noun + adjective rather than noun + verb.

Reply:
@adamchainz I'm not a native speaker so I'll defer to you but it sounded weird to me to have this method named `upload_stopped` while the other ones are not using this form (`file_complete`, `upload_complete` instead of `file_completed`, `upload_completed`).

Reply:
I read them as e.g. "when the upload is complete". In present tense it would be "the upload completes".